#goal definition
string mac
string token
nautonomous_operation_action/OperationPlan[] operationPlan
---
#result definition
nautonomous_operation_action/MissionStatus result
---
#feedback
nautonomous_operation_action/MissionStatus feedback